menu.js

```diff
@@ -19,6 +19,16 @@ function sendAction(action, data) {
 
 const viewSubmenu = [
 	{
+		label: 'Toggle Dark Mode',
```

Comment:
In Caprine we have this in the About menu. Really not sure what's the best location.